src/cargo/core/compiler/timings/report.rs

```diff
@@ -456,18 +462,24 @@ fn aggregate_sections(unit_time: &UnitTime) -> AggregatedSections {
                 SectionName::Frontend,
                 SectionData {
                     start: 0.0,
-                    end: rmeta,
+                    end: round(rmeta),
+                },
+            ),
+            (
+                SectionName::Codegen,
+                SectionData {
+                    start: round(rmeta),
+                    end: round(end),
                 },
             ),
-            (SectionName::Codegen, SectionData { start: rmeta, end }),
         ])
     } else {
         // We only know the total duration
         AggregatedSections::OnlyTotalDuration
     }
 }
 
-/// Rounds time in seconds to centisecond precision (0.01s).
+/// Rounds seconds to 0.01s precision.
```

Comment:
nit: Technically this should be on the previous commit

Reply:
Thanks for the review! Renamed and moved to the right commit.